parm/config/config.base.emc.dyn

```diff
@@ -47,10 +47,11 @@ export NOSCRUB="@NOSCRUB@"
 export BASE_GIT="@BASE_GIT@"
 
 # Toggle to turn on/off GFS downstream processing.
-export DO_BUFRSND="NO"    # BUFR sounding products
+export DO_BUFRSND="NO"     # BUFR sounding products
 export DO_GEMPAK="NO"      # GEMPAK products
 export DO_AWIPS="NO"       # AWIPS products
 export WAFSF="NO"          # WAFS products
+export DO_VRFY="YES"       # VRFY step
```

Comment:
Why is a DO_VRFY toggle added to the system? The vrfy job runs more than just verification (VSDB), it also runs the GSI monitoring package, tracker, and fit2obs. If it's set to YES by default that is fine, most users will want it on...but this toggle isn't needed right now. We'll be breaking apart the vrfy job in the near future so this toggle will likely be replaced by other ones for the individual pieces. Seeing how you implemented this I am not opposed but it's not needed at this time. Do the tracker, monitoring package, and fit2obs work via the vrfy job on Jet?

Reply:
Ok, if it's about turning off the METP jobs then the DO_METP variable in config.base should be set to NO when on Jet (it's currently YES by default everywhere). The METplus jobs are separate to the vrfy job. A new do_metp variable (similar to do_vrfy) in the setup scripts and configs should be added to turn off the gfsmetp jobs on Jet.

If the tracker and fit2obs pieces aren't yet working on Jet then yes we will want the vrfy job off and we can use the DO_VRFY switch you added. Please confirm those two pieces aren't yet working on Jet, thanks!

Reply:
tracker and fit2obs are not part of this PR. Please open another issue as outlined in PR standard.